core/osrd-railjson/src/main/java/fr/sncf/osrd/railjson/schema/rollingstock/RJSEffortCurves.java

```diff
@@ -33,6 +33,8 @@ public static final class RJSConditionalEffortCurve {
 
     public static final class RJSEffortCurveConditions {
         public RJSComfortType comfort = null;
+        @Json(name = "electrical_profile_level")
+        public String electricalProfileLevel = null;
```

Comment:
how about just electrical_profile?

Reply:
We are checking against the value of the electrical profile here, not against an electrical profile instance (which includes `trackRanges` and such)
In this regard do you think maybe `electrical_profile_value` would be clearer ?

Reply:
whatever, doesn't matter. I've always thought that "electrical profile" = value and "electrical profile target" = category of trains it applies to